tests/test_bibliotheca.py

```diff
@@ -67,12 +76,25 @@ def setup(self):
         self.collection = MockBibliothecaAPI.mock_collection(self._db)
         self.api = MockBibliothecaAPI(self._db, self.collection)
 
+    base_path = os.path.split(__file__)[0]
+    resource_path = os.path.join(base_path, "files", "bibliotheca")
+
     @classmethod
     def sample_data(self, filename):
         return sample_data(filename, 'bibliotheca')
+    @classmethod
+    def get_data(cls, filename):
+        path = os.path.join(cls.resource_path, filename)
+        return open(path).read()
```

Comment:
get_data and sample_data do the same thing -- you should be able to remove get_data and change all the references.